migrate/20231124_bdev_ubi.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
```

Comment:
If you put the migrations to a separate (earlier) commit, you can do deployments without breaking the control plane by following these steps;

- Deploy up to and including migration commit
- Perform database migration
- Deploy rest of the commits

If the migration is dropping some columns/tables instead of adding them, you can do something similar by putting migrations to a later separate commit.